Experimental/DistributedAuthoritySample/Assets/Scenes/CharacterTest.unity

```diff
@@ -0,0 +1,671 @@
+%YAML 1.1
```

Comment:
Analogously to the testing locomotion script, let's move this to a Temp (or Testing? Test is a reserved path for other purposes) subdirectory inside of Scenes folder.